ext/spl/tests/arrayObject_getFlags_basic2.phpt

```diff
@@ -7,10 +7,10 @@ var_dump($ao->getFlags());
 
 $ao2 = new ArrayObject($ao);
 var_dump($ao2->getFlags());
-var_dump($ao2->getIterator()->getFlags());
```

Comment:
I'd prefer to just switch off E-DEPRECATED in ini settings instead of using `error_reporting(0)`. Is there a problem in fixing the test cases to avoid the deprecation warning or just silencing E_DEPRECATED?

Reply:
Ignoring E_DEPRECATED is fine with me.